/**:
  ros__parameters:
    stereo_img_topic: /zed/zed_node/stereo/image_rect_color
    right_output_topic: /right_image
    left_output_topic: /left_image
    use_sim_time: false